app/storybook/package.json

```diff
@@ -2,28 +2,43 @@
   "name": "ui-kit-storybook",
   "private": true,
   "dependencies": {
-    "@emotion/css": "^11.10.5",
-    "@propeldata/ui-kit": "workspace:^",
+    "@emotion/react": "^11.11.1",
+    "axios": "^1.5.0",
```

Comment:
Why adding axios? Can't we use fetch instead?

Reply:
Please check my comments below